go/settings.py

```diff
@@ -314,6 +314,10 @@ def static_paths(paths):
         'task': 'go.scheduler.tasks.poll_tasks',
         'schedule': crontab(minute='*/5'),
     },
+    'update-daily-conversation-metrics': {
+        'task': 'go.vumitools.tasks.send_recent_conversation_metrics',
```

Comment:
I have a feeling that we should put `tasks` into `go.base` instead of `go.vumitools` because `go.vumitools` is meant to be things that don't depend on Django modules and `tasks` need Django and Celery. Likely it'll work as is and there is some advantage to it being closer to `go.vumitools.middleware`, but I think it's probably best not to break our no-django-in-go-vumitools rule.